lib/rules/declaration-empty-line-before/__tests__/index.mjs

```diff
@@ -629,6 +629,54 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['always', { except: ['after-block'] }],
+	fix: true,
+	computeEditInfo: true,
+	accept: [
+		{
+			code: 'a {\n\n top: 15px;\n}',
+		},
+		{
+			code: 'a {\n foo {} \n bottom: 5px;}',
+		},
+		{
+			code: 'a {\n @media foo {} \n bottom: 5px;}',
+		},
+		{
+			code: 'a {\r\n foo {} \r\n bottom: 5px;}',
+		},
+		{
+			code: 'a {\r\n @media foo {} \r\n bottom: 5px;}',
```

Comment:
```suggestion
			code: 'a {\n\n color: red;\n}',
		},
		{
			code: 'a {\n a {} \n color: red;}',
		},
		{
			code: 'a {\n @media all {} \n color: red;}',
		},
		{
			code: 'a {\r\n a {} \r\n color: red;}',
		},
		{
			code: 'a {\r\n @media all {} \r\n color: red;}',
```

To [follow our conventions](https://stylelint.io/developer-guide/rules#write-tests) for newly added tests.

The intention of doing this is that reviewers and maintainers know that the _contents_ of the selector, at-rule, prelude and declaration are irrelevant to the rule. It helps when scanning through lots of tests.